chain/client/src/client.rs

```diff
@@ -1053,6 +1053,8 @@ impl ClientActor {
         let prev_hash = head.last_block_hash;
         let prev_prev_hash = prev.inner.prev_hash;
 
+        debug!("Producing block at height {}, I'm {:?}", next_height, block_producer.account_id);
```

Comment:
add `target: "client"` here and below
I also generally prefer "{account_id} the rest" format so it's easier to see messages from different nodes.